scrapers/getWorldometers.js

```diff
@@ -34,13 +34,21 @@ const mapRows = (_, row) => {
 	const country = { updated: Date.now() };
 	cheerio(row).children('td').each((index, cell) => {
 		cell = cheerio.load(cell);
-		if (index === 0) {
-			const countryInfo = countryUtils.getCountryData(cell.text().replace(/(\n|,)/g, ''));
-			country.country = countryInfo.country || cell.text().replace(/(\n|,)/g, '');
-			delete countryInfo.country;
-			country.countryInfo = countryInfo;
-		} else if (columns[index - 1]) {
-			country[columns[index - 1]] = parseInt(cell.text().replace(/(\n|,)/g, '')) || 0;
+		// console.log(JSON.stringify(cell.text()));
+		switch (index) {
+			case 0: {
+				const countryInfo = countryUtils.getCountryData(cell.text().replace(/(\n|,)/g, ''));
+				country.country = countryInfo.country || cell.text().replace(/(\n|,)/g, '');
+				delete countryInfo.country;
+				country.countryInfo = countryInfo;
+				break;
+			}
+			case 12:
+				country[columns[index - 1]] = cell.text();
+				break;
+
+			default:
+				country[columns[index - 1]] = parseInt(cell.text().replace(/(\n|,)/g, '')) || 0;
```

Comment:
- [ ] remove console.log

- [ ]  change switch to if
`if (index === 0) {
                                 const countryInfo = countryUtils.getCountryData(cell.text().replace(/(\n|,)/g, ''));
				country.country = countryInfo.country || cell.text().replace(/(\n|,)/g, '');
				delete countryInfo.country;
				country.countryInfo = countryInfo;
} else {
                                  const value = cell.text();
                                  country[columns[index - 1]] = isNan(value) ? value : (parseInt(value.replace(/(\n|,)/g, '')) || 0);
}`

Reply:
Hey dude but switch is better than if, if setence evaluate one by one and switch evaluate once directly to the code block

Reply:
kinda true yes, let's leave it